.ci.yaml

```diff
@@ -12,12 +12,15 @@ platform_properties:
   linux:
     properties:
       os: Linux
+      signing_certs: "none"
   mac:
     properties:
       os: Mac-10.15
+      signing_certs: "none"
   windows:
     properties:
       os: Windows
+      signing_certs: "none"
```

Comment:
We only need to add this dimension for `Mac` bots.

Reply:
Thanks for the comments! I think if I add it as a common property it will apply to all luci builders?

Reply:
I was thinking about adding the dimension in the .ci.yaml, but `common` sounds easier to proceed. In that case, no extra work is needed to separate different platforms but the shared `none`. I am okay with `none` the default for all platforms.

Reply:
sounds good! pushed up https://flutter-review.googlesource.com/c/infra/+/29800 (this infra change will wait after this week's flutter 3 release). I will abort the changes on .ci.yaml files for cocoon/engine/framework, and use common.star to request luci builders to auto populate the properties to ci.yaml.